src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.js

```diff
@@ -0,0 +1,4 @@
+const checkExternalIdOverrides = externalId =>
+  externalId === 'bbc_oromo_radio' ? `bbc_afaanoromoo_radio` : externalId;
```

Comment:
This should be in the service config for `afaanoromoo` and used if set

Reply:
IMO overkill...

I'll do it though